src/components/App/App.js

```diff
@@ -24,21 +22,51 @@ const customStyles = {
     overflow: 'hidden',
     transform: 'translate(-50%, -50%)',
     backgroundSize: 'cover',
-    backgroundRepeat: 'no-repeat'
+    backgroundRepeat: 'no-repeat',
   },
   overlay: {
-    background: 'rgba(0, 0, 0, 0.5)'
-  }
+    background: 'rgba(0, 0, 0, 0.5)',
+  },
 };
 
 // Make sure to bind modal to your appElement (https://reactcommunity.org/react-modal/accessibility/)
-Modal.setAppElement('#root')
+Modal.setAppElement('#root');
 
 function App() {
-  const [allMovies, setMovies] = useState(movieData.movies);
+  const [allMovies, setMovies] = useState([]);
   const [focusMovie, setFocusMovie] = useState([]);
   // modal
   const [modalIsOpen, setIsOpen] = React.useState(false);
+  const [error, setError] = useState('');
+
+  function getAllMovies() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => {
+        if (!response.ok) {
+          console.log(`Error code: ${response.status}`);
+          throw new Error(`Sorry the Movies are not available`);
```

Comment:
Are you going to throw errors consistently, or use state to let the user know about errors? Pick one and stick with it for consistency!